### PR TITLE
Update cq sdk version

### PR DESCRIPTION
**Pull request by bbernays**
Implements #26 

- [x] ~~restructuring the resources to sub-packages as in other providers~~ DO doesn't have services so all resources are staying in a single package
- [x] use the new TestResource SDK function
- [x] ~~splitting tests into mock and integration tests~~ there are no mock tests
- [ ] re-writing all terraform files to use modules.

### CODE REVIEW

File: docs/docs.go [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+	"github.com/cloudquery/cq-provider-digitalocean/resources/provider"
 	"github.com/cloudquery/cq-provider-sdk/provider/docs"
 )
 
 func main() {
-	err := docs.GenerateDocs(resources.Provider(), "./docs")
+	err := docs.GenerateDocs(provider.Provider(), "./docs", true)
 	if err != nil {
-		fmt.Sprintf("Failed to geneerate docs: %s", err)
+		log.Fatalf("Failed to generate docs: %s", err)
```
Comment (bbernays) [inline docs/docs.go:13]: Bringing this in line with other providers

File: resources/account_test.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+package resources
+
+import (
+	"testing"
+
+	"github.com/cloudquery/cq-provider-digitalocean/client"
+)
+
+func TestIntegrationAccount(t *testing.T) {
+	client.DOTestHelper(t, Account(), "./snapshots")
```
Comment (yevgenypats) [inline resources/account_test.go:13]: we are not using snapshots anymore so I think this parameter is unused (I think you can search and replace/delete).

File: .github/workflows/int_test.yml [hunk 3]
```diff
@@ -48,7 +48,7 @@ jobs:
         run: go build -v .
 
       - name: Integration Test
-        run: go test -v -p 20 ./resources/integration_tests
+        run: go test -tags=integration ./...
```
Comment (yevgenypats) [inline .github/workflows/int_test.yml:51]: I'd add a timeout here (will test first how long it takes and will add this +1 min `-timeout=2min` or something like that)

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Looks good. few minor comments.
The terraform files can be added in this or follow-up. I don't know if DO has a lot of modules so maybe the only thing that will be necessary is just removing the whole `${var.test_prefix}` and `${var.test_suffix}` from  the tf files and reapply them.

File: resources/account_test.go (hunk 2 above)
Comment (bbernays) [reply resources/account_test.go]: Removed

File: .github/workflows/int_test.yml (hunk 3 above)
Comment (bbernays) [reply .github/workflows/int_test.yml]: Fixed

**Comment by bbernays** [comment]
> Looks good. few minor comments. The terraform files can be added in this or follow-up. I don't know if DO has a lot of modules so maybe the only thing that will be necessary is just removing the whole `${var.test_prefix}` and `${var.test_suffix}` from the tf files and reapply them.

I have addressed all of the comments. I also scheduled a few min for us to discuss the tf modules tomorrow so I will handle that in a second pr

**Pull request merged**